sparkmagic/sparkmagic/utils/startupinfo.py

```diff
@@ -0,0 +1,20 @@
+from sparkmagic.utils.utils import get_sessions_info_html
+
+class StartupInfoDisplay:
+    def __init__(self, ipython_display, sessions_info, current_session_id):
+        self.ipython_display = ipython_display
+        self.sessions_info = sessions_info
+        self.current_session_id = current_session_id
+
+    def write_msg(self, msg):
+        pass
+
+    def display_info(self):
+        pass
```

Comment:
I found it confusing that `defaultProgressIndicator` calls `display` in its `init` function, but the `defaultStartupInfoDisplay` has `display_info` method that is called after initialization.

Can we consolidate on one pattern? What do you think?